rviz_default_plugins/src/rviz_default_plugins/transformation/tf_wrapper.cpp

```diff
@@ -138,7 +139,22 @@ void TFWrapper::initialize(
 void TFWrapper::initializeBuffer(
   rclcpp::Clock::SharedPtr clock, rclcpp::Node::SharedPtr node, bool using_dedicated_thread)
 {
-  buffer_ = std::make_shared<tf2_ros::Buffer>(clock);
+  rcl_interfaces::msg::ParameterDescriptor descriptor;
+  descriptor.description = "Configure the rviz tf buffer cache time [ns].";
+  descriptor.read_only = true;
+  auto cache_time_ns = node->declare_parameter<int64_t>(
+    "tf_buffer_cache_time_ns",
+    std::chrono::duration_cast<std::chrono::nanoseconds>(
```

Comment:
I might suggest that this be `tf_buffer_cache_time_ms`, rather than `ns`.  That is, nanoseconds seems like too fine a resolution; I don't think users will care whether it takes 5 or 10 ns, but they might care about 10 or 50 ms.

Reply:
yeah that sounds fine to me

Reply:
Addressed in https://github.com/ros2/rviz/pull/792/commits/82896d3c87f2b0f3f83ce2880f37aa306c7f8412